lib/internal/Magento/Framework/Stdlib/DateTime/Filter/DateTime.php

```diff
@@ -12,13 +13,19 @@
  */
 class DateTime extends Date
 {
+    /**
+     * @var Resolver
+    */
+    private $localeResolver;
+
     /**
      * @param \Magento\Framework\Stdlib\DateTime\TimezoneInterface $localeDate
      *
      */
-    public function __construct(\Magento\Framework\Stdlib\DateTime\TimezoneInterface $localeDate)
+    public function __construct(\Magento\Framework\Stdlib\DateTime\TimezoneInterface $localeDate, Resolver $localeResolver)
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.4/contributor-guide/backward-compatible-development/)